Polynomials.Core/Polynomial.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Polynomials.Core
+{
+    public class Polynomial:IEquatable<Polynomial>
```

Comment:
Please add spaces before and after the colon

Reply:
Done.